qgsquick/from_qgis/plugin/editor/qgsquickeditorwidgetcombobox.qml

```diff
@@ -78,4 +79,26 @@ ComboBox {
     }
   }
   // [/hidpi fixes]
+
+  indicator: Item {
+    anchors.fill: parent
+
+    Image {
+      id: comboboxIndicatorIcon
+      source: QgsQuick.Utils.getThemeIcon("ic_angle_double_down")
+      height: parent.height * 0.4
+      width: height
+      anchors.right: parent.right
+      anchors.rightMargin: 10
```

Comment:
Please, dont forgot to use `QgsQuick.Utils.dp` for pixel sizes. Anyway, also would be good to check it with other widgets. Yesterday's [PR](https://github.com/lutraconsulting/input/pull/1189) introduces some margin variables for unified design.